nupic/__init__.py

```diff
@@ -19,3 +19,5 @@
 # http://numenta.org/licenses/
 # ----------------------------------------------------------------------
 __version__ = "0.2.4.dev0"
+
+NUPIC_ROOT = os.environ.get('NUPIC', os.path.dirname(os.path.realpath(__file__)))
```

Comment:
You need to `import os` in this file. See failing build jobs: https://travis-ci.org/numenta/nupic/builds/66349699